packages/article-converter/src/plugins/types.ts

```diff
@@ -20,6 +20,7 @@ export interface TransformOptions {
   previewAlt?: boolean;
   frontendDomain?: string;
   components?: DynamicComponents;
+  articleLanguage?: string;
```

Comment:
Burde kanskje bruke samme navn på denne overalt? `articleLanguage`/`locale`/`lang`?

Reply:
Sikkert lurt. articleLanguage og locale kan fort være to forskjellige ting.